09-student_guide.Rmd

```diff
@@ -0,0 +1,277 @@
+spike.phydat <- read.phyDat("sars_spike_protein.fasta", format = "fasta")
+
+spike.phydat
+```
+
+Different site patterns refers to sites that differ between sequences. In this small dataset, 46 of the 3827 possible bases (characters) show differences among these SARS-CoV-2 samples.
+
+We've been working with a dataset that contains the original SARS-CoV-2 sequence (the Wuhan reference sample), as well as samples of the alpha, beta, delta, and gamma variants. Another variant, the omicron variant, was first identified in late 2021 and quickly became a variant of concern. Let's look at a dataset that contains additional omicron samples.
+
+```{r, message = FALSE, warning = FALSE}
+spike.omicron <- read.phyDat("sars_spike_protein_omicron.fasta", format = "fasta")
+
+spike.omicron
+```
+
+::: {.reflection}
+QUESTIONS:
+
+1. What is some information saved in a .fasta object that RStudio tells us that we don't get from a .phyDat object?
+
+2. How many omicron sequences are there in the second file (loaded into the object `spike.omicron`)?
+
+3. Do you think there is more variability in the omicron sequences than in other variants (alpha, beta, delta, and gamma)? Why or why not?
+:::
+
+## Exercise Three: How are variants related to the original SARS-CoV-2 reference strain?
+
+We'll work with both the smaller spike protein dataset (saved as  `spike.small`) and the spike protein dataset with omicron samples (saved as `spike.omicron`) in this activity. 
+
+### Building a neighbor-joining tree
+
+We can use `phyDat` objects to build a type of phylogenetic tree called a neighbor-joining tree. We can do this in two steps in RStudio using the `phangorn` package.
+
+```{r, message = FALSE, warning = FALSE}
+dist.small <- dist.ml(spike.phydat)
+nj.small <- nj(dist.small)
+```
+
+The first command (`dist.ml`) converts the genetic data into a distance matrix, while the second command (`nj`) uses the matrix to group taxa based on the genetic distance between them. We plot the object created by the `nj` command to visualize our phylogenetic tree.
+
+```{r, message = FALSE, warning = FALSE}
+plot(nj.small)
+```
+
+We can do the same for the larger dataset that includes omicron sequences.
+
+```{r, message = FALSE, warning = FALSE}
+dist.omicron <- dist.ml(spike.omicron)
+nj.omicron <- nj(dist.omicron)
+plot(nj.omicron)
+```
+
+When reading phylogenetic trees, it's important to remember that relatedness is determined by the number of nodes between two taxa. The delta variant is most closely related to the beta variant because they share a common node (where the branches join together). We always look at common ancestors of clades, so we essentially read ancestry on a tree from right (where the taxa are) to left. Thus, the delta variant is more closely related to the beta variant because the delta branch connects to a node shared by the beta branch before it connects to a node shared with the gamma branch or the Wuhan reference/alpha variant branch.
```

Comment:
What if you created an image where there are colored shapes on the nodes you are talking about, for reference? Could just do this in powerpoint and use ottrpal, nothing crazy with ggplot :)

Reply:
Ooooh, good idea!